ush/rocoto/rocoto_viewer.py

```diff
@@ -194,9 +194,10 @@ def load_produtil_pythonpath():
     PRODUTIL = collections.defaultdict(list) 
     PRODUTIL['hera'] = '/scratch1/NCEPDEV/global/glopara/svn/nceplibs/produtil/trunk/ush'
     PRODUTIL['luna']  = '/gpfs/hps3/emc/global/noscrub/emc.glopara/svn/nceplibs/produtil/trunk/ush'
-    PRODUTIL['tide']  = '/gpfs/td1/emc/global/save/emc.glopara/svn/nceplibs/produtil/trunk/ush'
-    PRODUTIL['gyre']  = '/gpfs/gd1/emc/global/save/emc.glopara/svn/nceplibs/produtil/trunk/ush'
-    try_clusters = ('hera','luna','tide','gyre')
+    PRODUTIL['surge']  = '/gpfs/hps3/emc/global/noscrub/emc.glopara/svn/nceplibs/produtil/trunk/ush'
```

Comment:
why repeat the same path for `luna` and `surge` as well as for `mars` and `luna`. 
Makes it error prone when the code is duplicated.
```
PRODUTIL['luna'] = PRODUTIL['surge'] = '/gpfs/hps3/emc/global/noscrub/emc.glopara/svn/nceplibs/produtil/trunk/ush'
```
Same for `mars` and `venus`

Reply:
Will ingest this suggestion in a separate issue, I have some other small edits to make. Thanks!